app/experimenter/templates/experiments/section_branches.html

```diff
@@ -5,19 +5,31 @@
 {% endblock %}
 
 {% block section_content %}
+{% if not experiment.is_multi_pref %}
   {% if experiment.pref_type %}
   <strong>Pref Type</strong> <p>{{ experiment.pref_type }}</p>
   {% endif %}
   {% if experiment.pref_branch %}
   <strong>Pref Branch</strong> <p>{{ experiment.pref_branch }}</p>
   {% endif %}
+{% endif %}
   {% for variant in experiment.variants.all %}
     <div class="border-left-grey pl-3 mb-4">
       <h5>{{ variant.ratio }}% {{ variant.name }}</h5>
-      {% if experiment.is_pref_experiment %}
+      {% if experiment.is_multi_pref %}
+        {% for preference in variant.preferences.all %}
+        
+          <div class="border-left-grey pl-3 ml-2"> 
+            <p><strong> Pref Type = <span class="text-info">{{ preference.pref_type }}</span></strong></p>
+            <p><strong> Pref Branch = <span class="text-info">{{ preference.pref_branch }}</span></strong></p>
```

Comment:
I wouldn't do these two like this....  but I think just leave this and we'll land it and we can sit and play with styles together.